pkg/kubelet/hosts_test_file1

```diff
@@ -0,0 +1,8 @@
+# hosts file for testing.
```

Comment:
Yes, it should be written to a temp directory, or you can mock the filesystem call. I don't have a strong opinion since many of existing unit tests write to disk already. I don't want test data checked in the non-testing directory though